src/components/shared/Card/CardStyled.js

```diff
@@ -105,13 +121,18 @@ const LinkContent = styled.div`
 
     }
 
-
     & img {
         width:154px;
-        height: 100%;
+        height:154px;
         border-radius: 0px 12px 13px 0px;
```

Comment:
Vale a pena testar o object-fit: cover; aqui e veja se afeta algo na proporção da imagen no preview, dependendo da resposta da API isso pode fazer muita diferença.
```suggestion
        border-radius: 0px 12px 13px 0px;
        object-fit: cover;
```